scrapers/superbemodels.yml

```
@@ -0,0 +1,29 @@
+name: superbemodels
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - superbemodels.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="item-name-model-video item-tags-change"]//h1
+      Date:
+         selector: //span[@class="duration-box duration-box-change"]//span[contains(.,"Release date:")]/em
+         postProcess:
+           - parseDate: January 2, 2006
+      Details: //div[@class="item description-video"]/p
+      Performers:
+        Name: //a[@class="models-name-box "]//a
```

`//a` of `//a` doesnt feel right. It seems to work though
The
```suggestion
        Name: //a[@class="models-name-box "]
```
is enough